### ISSUE TITLE
add ability to search public channels and show channel url

### ISSUE THREAD

**Issue by codegod100** [prefix-dev/pixi#475]
### Problem description

On the website if you do a search it searches through public channels, but if you use cli it doesn't.

Please make cli search public channels and show their urls for easy copy/paste

**Comment by ruben-arts** [comment]
I'm not entirely sure what you are requesting. 
I think you mean the `pixi add` command. But I'm not sure. The cli search searches through conda-forge or the channels defined in the `pixi.toml` when you run it in a project. 

Could you elaborate some more and maybe even give an example cli output?

**Comment by codegod100** [comment]
Yes it searches through the toml. My request is that it searches through channels not in the toml as well if they are public

**Comment by ruben-arts** [comment]
I'm not sure this is valuable as "all public" is not possible as we don't know this list. And if you just want to find the ones in prefix.dev then using the website would be a better experience. Closing for now because it was stale.
